### PR TITLE
Tackle TODO tasks

### PR DESCRIPTION
**Pull request by Avuidrauxs**
Handles the issue from https://github.com/GroceriStar/showcase/issues/171

### Description
- Moves inline styles in RecipeHeader and RenderContent to js objects
- Refactored `getLink` into `Router.js` and replaced usage of `href` with `Link` component

### CODE REVIEW

**Comment by atherdon** [comment]
ok, code looks fine, but we have build errors.
you can check details at outputs here: 
https://app.netlify.com/sites/boring-benz-76e55f/deploys/5c4028043ab7e4000929a771
https://travis-ci.org/GroceriStar/showcase/jobs/480734128

**Comment by Avuidrauxs** [comment]
@atherdon Sorry , it was a blatant oversight , build errors have been resolved now

**Comment by atherdon** [comment]
not a problem at all. this is why i love this robots and checks - they really save our time and make us less worried ;)

File: src/components/Cell/RecipeHeader.js [hunk 1]
```diff
@@ -43,40 +51,39 @@ const { Header, Footer, Content } = Layout;
...
+	// if (this.props.title !== nextProps.title || this.props.toggle !== nextProps.toggle) {
+	//   console.log("Name and Id RenderFooter **********************");
+	//       return true;
+	//     }
+	//     console.log("False RenderFooter****************************");
+	//     return false;
+	//   }
+	render() {
+		return (
+  <Header className='RecipeHeader-header'>
```
Comment (atherdon) [inline src/components/Cell/RecipeHeader.js]: this part is non formatted well, please update it

Comment (atherdon) [reply src/components/Cell/RecipeHeader.js]: does class RecipeHeader get updated styles? i mean as we removed a scss file? not sure...
and your approach of naming classes looks like BEM style. it's bad(not critical, but..). Details: https://medium.com/@bluebirdyourdestiny/bem-should-not-exist-6414005765d6

Comment (Avuidrauxs) [reply src/components/Cell/RecipeHeader.js]: Oooops , I have resolved it now.

Comment (atherdon) [reply src/components/Cell/RecipeHeader.js]: cool - please add it to this PR

Comment (atherdon) [reply src/components/Cell/RecipeHeader.js]: and sync your fork, because we have conflicts

**Comment by atherdon** [comment]
perfect - merging

**Pull request merged**